src/slack_annotations/format.py

```diff
@@ -82,17 +82,28 @@ def _build_annotation_summary(annotation: dict[str, Any]) -> str:
 
 
 def _build_annotation_fields(annotation: dict[str, Any]) -> list[dict[str, Any]]:
+    quote = None
     try:
         quote = _get_quote(annotation)
     except Exception:  # pylint:disable=broad-exception-caught
-        quote = NONE_TEXT
-
-    return [
-        {"type": "mrkdwn", "text": "*Quote:*"},
-        {
-            "type": "mrkdwn",
-            "text": f"*Annotation* (<{annotation['links']['incontext']}|in-context link>):",
-        },
-        {"type": "plain_text", "text": quote},
-        {"type": "plain_text", "text": _get_text(annotation)},
-    ]
+        pass
+    incontext_link = annotation["links"]["incontext"]
+
+    if annotation.get("references"):
+        fields = [
+            {
+                "type": "mrkdwn",
+                "text": f"*Reply* (<{incontext_link}|in-context link>):",
```

Comment:
Taking the same approach as in https://github.com/hypothesis/slack-annotations/pull/11#discussion_r1876195208.